app/repositories/PillarRepository.scala

```diff
@@ -0,0 +1,29 @@
+package repositories
+
+import model.Pillar
+import play.api.Logger
+import services.Dynamo
+
+import scala.collection.JavaConversions._
+
+
+object PillarRepository {
+  def getPillar(id: Long): Option[Pillar] = {
+    Option(Dynamo.pillarTable.getItem("id", id)).map(Pillar.fromItem)
+  }
+
+  def updatePillar(pillar: Pillar): Option[Pillar] = {
+    try {
+      Dynamo.pillarTable.putItem(Pillar.toItem(pillar))
+      Some(pillar)
+    } catch {
+      case e: Error =>
```

Comment:
Should probably be NonFatal or something? `Error` in java is usually for things like out of heap space which we can't really handle.

Reply:
true, though `Error` is currently used all over this app :)

Reply:
Yeah, looks like some mistakes were made 😞 

I'll add it to our health trello